app/api/route/thread_api.rb

```diff
@@ -7,6 +7,9 @@ class ThreadApi < Base
       optional :issue_id, type: Integer, desc: 'ID of issue'
       optional :order_by, type: Symbol, values:%i( created_at id  ), desc: 'Order of returned issues.'
       optional :order, type: Symbol, values:%i(asc desc), default: :asc
+      optional :end_date, type: Date, desc: 'No threads after the end date are returned'
+      optional :start_date, type: Date, desc: 'No threads before the start date are returned'
+      optional :datetime_less_than, type: DateTime, desc: 'No threads before the start date and time are returned'
```

Comment:
Can we just have 
```ruby
optional :before, type: [Date, DateTime], desc: : 'Only threads before this date are returned'
optional :after, type: [Date, DateTime], desc: : 'Only threads after or equal to this date are returned'
```

Reply:
I think, that the `[Date, DateTime]` doesn't work. I have got
```
{
  "error": "before is invalid"
}
```
for both date and datetime format on input.

Reply:
Aha, it is `types` not `type` in that case.

Reply:
:disappointed: sorry about that, I should have checked myself first.